src/__tests__/test_distillation_tower.js

```diff
@@ -1,18 +1,17 @@
 import MLR from '..';
 
 var fs = require('fs');
-var path = require('path');
 
 var Papa = require('papaparse');
 
 // Dataset -> https://openmv.net/info/distillation-tower
 
 describe('multivariate linear regression', () => {
   it('should work with large dataset with intercept', () => {
-    const ymatrix = fs.readFileSync(path.resolve(path.join('./data/ydistillationjs.csv')), 'utf8');
-    const y = Papa.parse(ymatrix, { delimiter: ',' }).data.map((a) => a.map((b) => parseFloat(b)));
-    const xmatrix = fs.readFileSync(path.resolve(path.join('./data/xdistillationjs.csv')), 'utf8');
-    const x = Papa.parse(xmatrix, { delimiter: ',' }).data.map((a) => a.map((b) => parseFloat(b)));
+    const ymatrix = fs.readFileSync('./data/ydistillationjs.csv', 'utf8');
```

Comment:
To read relative files use the following syntax

join(__dirname,'./data/ydistillationjs.csv')

So in your case 

 const ymatrix = fs.readFileSync(join(__dirname,'./data/ydistillationjs.csv'), 'utf8');

Reply:
@targos Is this correct ?

Reply:
it should be `join(__dirname, '../../data/ydistillationjs.csv')` (relative to the test file)

Reply:
thank you